components/skeleton/skeleton-mixin.js

```diff
@@ -167,7 +169,46 @@ export const SkeletonMixin = dedupeMixin(superclass => class extends RtlMixin(su
 
 	constructor() {
 		super();
-		this.skeleton = false;
+		this._skeleton = false;
+		this._skeletonActive = false;
+		this._skeletonWait = false;
+
+		this._parentSkeleton = new EventSubscriberController(this, 'skeleton', {
+			onSubscribe: this._onSubscribe.bind(this),
+			onUnsubscribe: this._onUnsubscribe.bind(this)
+		});
+	}
+
+	get skeleton() {
+		return this._skeletonActive;
+	}
+
+	set skeleton(val) {
+		const oldVal = this._skeleton;
+		if (oldVal === val) return;
+		this._skeleton = val;
+		this._skeletonActive = val;
```

Comment:
this seems wrong, but after we set this, we notify the parent that something changed, and they'll recalculate this value for us.

Reply:
I think a comment would be good here.  Does setting it to `undefined` do the same thing, forcing an update or does that break something?